src/combinator.rs

```diff
@@ -1134,17 +1134,4 @@ mod tests {
             .chain(just(','));
         assert_eq!(parser.parse("-,-,-,"), Ok(vec!['-', '-', '-', ',']))
     }
-
-    #[test]
-    // TODO: this fails
-    fn reproduce_repeated_with_infix() {
```

Comment:
Does this need a `#[should_fail]`?

Reply:
Yes, but I deleted the test in my latest commit.

Reply:
Apparently not.